tests/components/homematicip_cloud/test_climate.py

```diff
@@ -428,13 +431,62 @@ async def test_hmip_heating_group_heat_with_radiator(
     assert ha_state.attributes["min_temp"] == 5.0
     assert ha_state.attributes["max_temp"] == 30.0
     assert ha_state.attributes["temperature"] == 5.0
-    assert ha_state.attributes[ATTR_PRESET_MODE] is None
+    assert ha_state.attributes[ATTR_PRESET_MODE] == "Default"
+    assert ha_state.attributes[ATTR_PRESET_MODES] == [
+        PRESET_BOOST,
+        PRESET_ECO,
+        "Default",
+    ]
+
+
+async def test_hmip_heating_profile_naming(
```

Comment:
This test is unclear to me for several reasons:

1) I expect an arrange/act/assert structure for each test. This test asserts in between, it asserts on many unrelated things and it ends with an action that is never asserted upon.

2) I would also expect a test which checks that a default profile name also shows up nicely